### DISCUSSION TITLE
Scaling for entity pv_power_total

### DISCUSSION THREAD

**Discussion by chfink0** [wills106/homeassistant-solax-modbus#249] [Ideas]
Hello, 

great add-on, really so happy to have the Sofar (HYD 15KTL-3PH) Inverter finally in homeassitant. Awesome!

However, I mentioned that the scaling for pv_power_toal (register 05c4) is off by factor 10.  Changing the plugin_sofar.py from scale 0.01 to 0.1 did the trick. Maybe something you want to put into your release.

Thanks again for this great add-on!!

**Comment by wills106** [comment]
What version are you running? This was corrected in [2022.12.6](https://github.com/wills106/homeassistant-solax-modbus/releases/tag/2022.12.6)

**Reply by chfink0** [reply]
Thanks for your feedback, for whatever reason I have 2022.12.4. I installed on a brand new homeassistant instance 2 days ago via HACS as described in your documentation. Strange. I will update then to 2022.12.6. Thank you and sorry for the confusion then.

**Reply by wills106** [reply]
I probably released 2022.12.6 just after you had installed as it was only released 2 days ago.

Do you install via HACS? There is a sensor you can view to see if there are updates available.
You will find you can ignore some updates though as they might be specific to another Inverter brand / model, just check the release notes for each release.

**Reply by chfink0** [reply]
Yes, this must have been the case! I also saw the flag and message which offers me to update via HACS and also shows the relevant changes so I can decide to go up or stay. very nice!
